package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "black-hole-stream": "~0.0.1",
-    "busboy": "^0.2.8",
```

Comment:
https://github.com/cojs/busboy/blob/master/package.json#L25 already upgrade to latest version 1.6.0